src/init.cpp

```diff
@@ -778,6 +778,7 @@ void SetupServerArgs(NodeContext& node)
     argsman.AddArg("-daemon", "Run in the background as a daemon and accept commands", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
 #else
     hidden_args.emplace_back("-daemon");
+    hidden_args.emplace_back("-daemonwait");
```

Comment:
21447: we don't have 21007 (which introduced `-daemonwait`) yet, should be skipped for now